src/index.ts

```diff
@@ -17,16 +22,36 @@ const env = {
   location: process.env.ORGANIZATION ?? 'sarcophagus.aragonid.eth',
 }
 
+function generateVoteId(id: string): any {
+  const aragonVoteIdPrefix = "appAddress:0xf483c1f7858dd19915d0689d26cb3487fc90b640-vote:"
+  let aragonVoteId = aragonVoteIdPrefix + "0x" + Number(Number(id).toString(16)).toString(16)
+  return aragonVoteId
+}
+
 async function main() {
   const org = await connect(env.location, 'thegraph', { network: env.network })
   const voting = await connectVoting(org.app('voting'))
   const votes = await voting.votes({ first: 100 })
-  const votesWithCasts = await Promise.all(
-    votes.map(async (vote) => ({ ...vote, casts: await vote.casts() }))
-  )
-  
+
+  let voteId = generateVoteId(process.env.VOTE_ID)
```

Comment:
@solidoracle  --- There's a few more `let` variables that can be `const` --- my IDE flags these for me (which makes them easy to spot). These can get updated in a followup PR.